processor/src/test/java/org/mapstruct/ap/test/bugs/_1828/FirstMapper.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.bugs._1828;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public interface FirstMapper {
+
+    FirstMapper INSTANCE = Mappers.getMapper( FirstMapper.class );
+
+    @Mapping(target = "name", source = "name")
```

Comment:
is this required? name based mapping?

Reply:
Nope it isn't I copied it from the example